res/TemplateBatchFiles/libv3/models/protein/matrix2dict_aux.bf

```diff
@@ -94,89 +94,83 @@ models.protein.JTT.empirical_Q = {
+{0.062992709999999993, 0.33623259999999999, 0.039721729999999997, 0.033575769999999998, 0.0072131779999999998, 0.001233336, 0.076595659999999996, 0.021872639999999999, 2.298295, 10.96748, 5.6479850000000003, 1.238634, 0.0, 3.94794, 0.074172790000000002, 0.018613540000000001, 1.4156120000000001, 0.10111489999999999, 0.0022467590000000001, 4.3967200000000002}
+{0.11301460000000001, 0.082086770000000003, 0.19554460000000001, 0.1031734, 0.1993818, 0.00149661, 0.052886250000000003, 0.19847719999999999, 5.642309, 2.7147049999999999, 3.3906179999999999, 0.0046490350000000001, 3.94794, 0.0, 0.35561979999999999, 0.98130640000000002, 0.036744859999999997, 0.2199856, 7.0744639999999999, 1.6439459999999999}
+{1.800713, 0.3498713, 0.0073425540000000003, 0.1509482, 0.0048783949999999998, 0.74269090000000004, 0.028898150000000001, 0.079150559999999995, 10.494960000000001, 0.050165679999999997, 1.149931, 0.0099489939999999992, 0.074172790000000002, 0.35561979999999999, 0.0, 1.284651, 0.90571120000000005, 0.0055160740000000002, 0.19921330000000001, 0.2217442}
+{0.99883580000000005, 1.9264349999999999, 7.3483460000000003, 0.58229880000000001, 0.26394820000000002, 0.00059064049999999998, 0.067767090000000002, 0.99842149999999996, 5.4391160000000003, 0.60076070000000004, 0.1580539, 0.086884050000000004, 0.018613540000000001, 0.98130640000000002, 1.284651, 0.0, 3.0585749999999998, 0.1385142, 0.8104751, 0.074770409999999995}
+{2.9123169999999998, 1.1352580000000001, 2.1471749999999998, 0.15168809999999999, 3.225214e-06, 0.12020939999999999, 0.060166240000000003, 0.078627669999999997, 3.4432849999999999, 3.0871520000000001, 0.57027919999999999, 1.0392980000000001, 1.4156120000000001, 0.036744859999999997, 0.90571120000000005, 3.0585749999999998, 0.0, 0.01412361, 0.099842550000000002, 0.2166054}
+{0.079395489999999999, 0.57242859999999995, 0.00073109369999999996, 0.01423897, 0.44408330000000001, 4.3329830000000002e-05, 0.02252612, 0.13868530000000001, 7.01389, 0.063187480000000004, 0.3378544, 0.0080242630000000002, 0.10111489999999999, 0.2199856, 0.0055160740000000002, 0.1385142, 0.01412361, 0.0, 0.61212840000000002, 0.096635689999999996}
+{0.1433528, 0.17113149999999999, 2.622763, 0.90783380000000002, 0.77416119999999999, 0.027370909999999998, 0.1240642, 0.22958419999999999, 20.55414, 0.29031649999999998, 0.15213199999999999, 0.07109973, 0.0022467590000000001, 7.0744639999999999, 0.19921330000000001, 0.8104751, 0.099842550000000002, 0.61212840000000002, 0.0, 0.5010635}
+{3.7744770000000001, 0.1366145, 0.049312059999999998, 0.40760740000000001, 0.022435119999999999, 0.0090477370000000001, 0.57954090000000003, 0.42281999999999997, 6.890244, 7.9266750000000004, 3.59531, 0.034934399999999997, 4.3967200000000002, 1.6439459999999999, 0.2217442, 0.074770409999999995, 0.2166054, 0.096635689999999996, 0.5010635, 0.0}};
+
+
+
+
+
+
+fprintf (stdout, mapMatrixToDict (models.protein.alphabet, cpREV), "\n"):
```

Comment:
Is this batch file intended to be a utility script executed independently? It might be better to place the script elsewhere if that is the case.